stylesheets/partials/layouts/agency-homepage.scss

```diff
@@ -0,0 +1,138 @@
+/*ADD TO ANIMAL SERVICES FOLDER*/
```

Comment:
General question -- is this just cut and pasted from Sam's stuff? There's no nesting or anything going on here, and I think it's worth a quick refactoring to make it sassier.